src/controller/CHIPDevice.cpp

```diff
@@ -167,9 +168,37 @@ CHIP_ERROR Device::Deserialize(const SerializedDevice & input)
 void Device::OnMessageReceived(const PacketHeader & header, const PayloadHeader & payloadHeader,
```

Comment:
In the longer term, presumably we will do parsing of the response somewhere here instead of just handing it off to the application?  Especially for things like the attribute read....

Reply:
Yes, that's right. Need to add the response processing here eventually.